libs/vscode/messaging/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "@nx-console/vscode-messaging",
+  "version": "0.0.1",
+  "private": true,
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "nx": {
```

Comment:
nit but other libs have an nx name without the scope, so just `vscode-messaging` in this case